### PR TITLE
addfb payment support, add xcode build setting faq, add fb list and c…

### PR DESCRIPTION
**Pull request by jpjoyal**
…arousel guide doc

### CODE REVIEW

File: pages/guide/facebook-messenger.md [hunk 1]
```diff
@@ -48,6 +48,22 @@ If you are using our Stripe integration to power payments on other channels, ena
 
 The default currency is automatically selected when using the [button syntax](https://docs.smooch.io/guide/sending-images-and-buttons-shorthand/). When using the [Post Message API](https://docs.smooch.io/rest/#action-buttons) you can specify the currency for each call. However if that message needs to be delivered to Messenger, it will be rejected.
 
+### Carousel
+
+Messenger offer full support for carousels which allows you to send a set of horizontally scrollable items that combines text, image and any type of action buttons. 
+
+![Carousel example](http://blog.smooch.io/content/images/2016/08/carousel_demo.gif)
+
+You can send carousels containing up to 10 items via the [Post message API](https://docs.smooch.io/rest/#carousel-message)
```
Comment (dannytranlx) [inline pages/guide/facebook-messenger.md]: Maybe double check the copy of others. But I think `Post Message API` should have a capital `M` in `Post Message`.

File: pages/guide/facebook-messenger.md [hunk 2]
```diff
@@ -48,6 +48,22 @@ If you are using our Stripe integration to power payments on other channels, ena
 
 The default currency is automatically selected when using the [button syntax](https://docs.smooch.io/guide/sending-images-and-buttons-shorthand/). When using the [Post Message API](https://docs.smooch.io/rest/#action-buttons) you can specify the currency for each call. However if that message needs to be delivered to Messenger, it will be rejected.
 
+### Carousel
+
+Messenger offer full support for carousels which allows you to send a set of horizontally scrollable items that combines text, image and any type of action buttons.
```
Comment (mspensieri) [inline pages/guide/facebook-messenger.md]: offer**s** (add s)

> and any type of action buttons
reply and location actions are not usable in carousel

File: pages/guide/facebook-messenger.md [hunk 3]
```diff
@@ -48,6 +48,22 @@ If you are using our Stripe integration to power payments on other channels, ena
...
+
+Messenger offer full support for carousels which allows you to send a set of horizontally scrollable items that combines text, image and any type of action buttons. 
+
+![Carousel example](http://blog.smooch.io/content/images/2016/08/carousel_demo.gif)
+
+You can send carousels containing up to 10 items via the [Post Message API](https://docs.smooch.io/rest/#carousel-message)
+
+### List
+
+Messenger also has full support for list messages. List messages are a vertically scrollable set of items that may each contain text, an image, and action button.
```
Comment (mspensieri) [inline pages/guide/facebook-messenger.md]: > and action button.
and **an** action button.

Maybe also mention the "default" action? Technically there can be two actions per item

Comment (jpjoyal) [inline pages/guide/facebook-messenger.md]: "Technically there can be two actions per item" what ?

Comment (mspensieri) [inline pages/guide/facebook-messenger.md]: There is a button on the list item, and there is also a default action that happens when you tap the cell. This means in Smooch a list object can have two actions in the array, one button and one on cell tap.

Ex:
User sends a list of products. Each one has a buy button. Tapping on the cell opens a webview with more information about the product (a link action)

**Pull request merged**